src/main/kotlin/solve/settings/visualization/popover/SettingsDialogNode.kt

```diff
@@ -0,0 +1,19 @@
+package solve.settings.visualization.popover
+
+import javafx.scene.layout.VBox
+import solve.styles.Style
+import tornadofx.*
+
+class SettingsDialogNode : VBox(10.0) {
+
+    var title: String = ""
+
+    fun addTitle() {
```

Comment:
Why it is not added from the start?

Reply:
Because the title contains the name of the markup file. Title is added when the type of landmark is already known